### PR TITLE
Added text-typography project

### PR DESCRIPTION
**Pull request by Naufil-Shemle**
**Please review my code with your valuable approval.**

### CODE REVIEW

File: tasks-by-users/Naufil-Shemle/07-text-typography/style.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
+* {
+  margin: 0;
+  padding: 0;
```
Comment (faran4engg) [inline tasks-by-users/Naufil-Shemle/07-text-typography/style.css:3]: missing `box-sizing`

**Review by faran4engg — APPROVED** [review]

**Comment by Naufil-Shemle** [comment]
**Please also try to review this PR as soon as possible.**

**Review by Naufil-Shemle — COMMENTED** [review]
**Please also try to review this PR as soon as possible.**

**Review by mmusk — APPROVED** [review]
LGTM

**Review by anask601 — APPROVED** [review]

**Pull request merged**